src/selectors/entities/posts.js

```diff
@@ -391,3 +391,15 @@ export const getMostRecentPostIdInChannel = createSelector(
         return postIdsInChannel[0];
     }
 );
+
+export const getLatestReplyablePost = createSelector(
+  getPostsInCurrentChannel,
```

Comment:
It's hard to tell here, but can you make sure this is indented with 4 spaces? It looks like it might only have 2. I'm also not sure why Jenkins wouldn't be complaining if that was the case though

Reply:
Ya, true it had just two spaces. Fixed it. Gave a PR for fixing the eslint rule as well. 
https://github.com/mattermost/mattermost-redux/pull/325/files